spec/OpenLineage.yml

```diff
@@ -0,0 +1,250 @@
+openapi: 3.0.2
+info:
+  title: OpenLineage
+  version: 0.0.1
+  description: OpenLineage is an open source **lineage and metadata collection API** for the data ecosystem.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+paths:
+  /events:
```

Comment:
We might want to contextualize the endpoint a bit more. Meaning, `events` is rather generic and can mean all types of events (not just specific to open lineage). I'd also expect to the payload structured as an array of different event types. That said, I understand the endpoints overall intent: _capture event **X** at a given time **T** with metadata observed_.

I think some alternative endpoints can be:

-  `/metadata`: Yeah, still "generic", but now conveys that the payload will contain objects / properties specific to metadata collection
-  `/lineage`: Collect metadata specific to data lineage, mapping the objects / properties in the payload to the domain of open lineage.

Reply:
`/lineage` makes sense to me, I'll update accordingly.